src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculate {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private String color = colorSupplier.getRandomColor();
```

Comment:
lets init values from outside
exact FigureSupplier should generate figures with random properties:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/73041091/ce46f7a7-b894-46cb-8d3f-d171679c653c)
```suggestion
```